app/css/background-colors.css

```diff
@@ -19,3 +19,13 @@
 .bg-fuchsia { background-color: var(--fuchsia) }
 .bg-purple  { background-color: var(--purple) }
 .bg-maroon  { background-color: var(--maroon) }
+
+.bg-warning-stripes {
```

Comment:
Hah! Actually I think that's kinda the solution. Just pushed a new `backgrounds.css`